backend/dao-service/src/proposal-service/proposal-service.service.ts

```diff
@@ -39,17 +39,13 @@ export class ProposalServiceService {
   }
 
 
-  async findOne(id: number, sub: string): Promise<any> {
+  async findAllProposals(sub: string): Promise<any> {
     const role = await this.getUserRole(sub);
     console.log(role);
```

Comment:
Try to use logger. Avoid console log in Dev merge

Reply:
This has been fixed